miic-mapping.csv

```diff
@@ -0,0 +1,331 @@
+"Td/Tdap","pertussis vaccine",,,"pertussis vaccine",,"11","Inactive"
+"Td/Tdap","Td, unspecified formulation",,,"Td(adult) unspecified formulation",90714,"139","Inactive"
+"Td/Tdap","tetanus diphtheria not adsbed adult",,,"tetanus and diphtheria toxoids, not adsorbed, for adult use",,"138","Inactive"
+"Td/Tdap","DT, IPV adsorbed (non-US)",,,"Diphtheria, Tetanus, Poliomyelitis adsorbed (non-US vaccine)",,195,"Inactive"
+"Td/Tdap","Td (adult), adsorbed",,,"tetanus and diphtheria toxoids, adsorbed, preservative free, for adult use, Lf unspecified",,196,"Active"
+"Tetanus","Tetanus toxoid, adsorbed","TT adsorbed","PMC","tetanus toxoid, adsorbed",90703,"35","Inactive"
+"Tetanus","Tetanus toxoid, not adsorbed","TT not adsorbed",,"tetanus toxoid, not adsorbed",,"142","Inactive"
+"Tetanus","tetanus toxoid, unspecified formulation",,,"tetanus toxoid, unspecified formulation",,"112","Inactive"
+"Tick-borne Encephalitis","Tick-borne encephalitis (non-US)",,,"Tick-borne encephalitis (non-US)",,77,"Inactive"
+"Tick-borne Encephalitis","Tick-borne encephalitis, unspecified",,,"Tick-borne encephalitis, unspecified",,222,"Active"
+"Tick-borne Encephalitis","Tick-borne encephalitis, inact, PF, 0.25mL","TICOVAC peds","PFR","Tick-borne encephalitis, inact, PF, 0.25mL",90626,223,"Active"
+"Tick-borne Encephalitis","Tick-borne encephalitis, inact, PF, 0.5mL","TICOVAC","PFR","Tick-borne encephalitis, inact, PF, 0.5mL",90627,224,"Active"
+"Typhoid","Typhoid-ViCPs","Typhim Vi","PMC","typhoid Vi capsular polysaccharide vaccine",90691,"101","Active"
+"Typhoid","Typhoid-HP","Typhoid",,"typhoid vaccine, parenteral, other than acetone-killed, dried",90692,"41","Inactive"
+"Typhoid","Typhoid-AKD","Typhoid-AKD",,"typhoid vaccine, parenteral, acetone-killed, dried (U.S. military)",90693,"53","Inactive"
+"Typhoid","Typhoid-Oral","Vivotif B","CRU","typhoid vaccine, live, oral",90690,"25","Active"
+"Typhoid","Typhoid, unspecified formulation",,,"typhoid vaccine, unspecified formulation",,"91","Inactive"
+"Typhus","Typhus, historical","Typhus",,"Historical record of a typhus vaccination",86000,"131","Inactive"
+"Varicella","MMRV","Proquad","MSD","measles, mumps, rubella, and varicella virus vaccine",90710,"94","Active"
+"Varicella","Varicella","Varivax","MSD","varicella virus vaccine",90716,"21","Active"
+"Yellow Fever","Yellow Fever, alt","Stamaril","PMC","Yellow fever vaccine alternative formulation",90717,"183","Active"
+"Yellow Fever","Yellow Fever","YF-VAX","PMC","yellow fever vaccine",90717,"37","Active"
+"Yellow Fever","Yellow Fever, unspecified",,,"Yellow fever vaccine, unspecified formulation",,"184","Inactive"
+"Zoster/shingles","zoster subunit","Shingrix","SKB","zoster vaccine subunit",90750,"187","Active"
+"Zoster/shingles","Zoster vaccine, live","Zostavax","MSD","zoster vaccine, live",90736,"121","Active"
+"Meninge-conj","MenABCWY, PF","PENBRAYA","PFR","meningococcal polysaccharide (MenACWY-TT conjugate), (MenB), PF",,316,"Active"
+,,,,,,,
```

Comment:
hmm. Not sure if this breaks anything at the moment. (also, I ran the rake task in the Rails app and it produced the same thing)

Reply:
It shouldn't since there's no corresponding CVX code.